FE exits with error:"failed to read after retried 3 times!"

<!-- (At least include the following, feel free to add if you have more content) -->
3FE.
### Steps to reproduce the behavior (Required)

### Expected behavior (Required)

### Real behavior (Required)
```
2022-06-28 15:18:43,739 WARN (replayer|76) [GlobalStateMgr.replayJournal():1708] got exception when get next, will exit,
com.starrocks.journal.JournalException: failed to read after retried 3 times! key = 605868, db = CloseSafeDatabase{db=593077}
        at com.starrocks.journal.bdbje.BDBJournalCursor.next(BDBJournalCursor.java:226) ~[starrocks-fe.jar:?]
        at com.starrocks.server.GlobalStateMgr.replayJournal(GlobalStateMgr.java:1706) [starrocks-fe.jar:?]
        at com.starrocks.server.GlobalStateMgr$4.runOneCycle(GlobalStateMgr.java:1505) [starrocks-fe.jar:?]
        at com.starrocks.common.util.Daemon.run(Daemon.java:115) [starrocks-fe.jar:?]
Caused by: com.sleepycat.je.LockTimeoutException: (JE 7.3.7) Lock expired. Locker 413203543 -1_replayer_ReplicaThreadLocker: waited for lock on database=593077 LockAddr:1880362241 LSN=0x4c/0x89dcff type=READ grant=WAIT_NEW timeoutMillis=1000 startTime=1656400722739 endTime=1656400723739
Owners: [<LockInfo locker="<ReplayTxn id="-606084">2016626435 -606084_ReplayThread_ReplayTxn" type="WRITE"/>]
Waiters: [<LockInfo locker="413203543 -1_replayer_ReplicaThreadLocker" type="READ"/>]

        at com.sleepycat.je.txn.LockManager.makeTimeoutException(LockManager.java:1117) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.txn.LockManager.waitForLock(LockManager.java:606) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.txn.LockManager.lock(LockManager.java:345) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.txn.BasicLocker.lockInternal(BasicLocker.java:124) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.rep.txn.ReplicaThreadLocker.lockInternal(ReplicaThreadLocker.java:63) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.txn.Locker.lock(Locker.java:499) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.dbi.CursorImpl.lockLN(CursorImpl.java:3585) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.dbi.CursorImpl.lockLN(CursorImpl.java:3316) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.dbi.CursorImpl.lockLNAndCheckDefunct(CursorImpl.java:2138) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.dbi.CursorImpl.searchExact(CursorImpl.java:1950) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.Cursor.searchExact(Cursor.java:4194) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.Cursor.searchNoDups(Cursor.java:4055) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.Cursor.search(Cursor.java:3857) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.Cursor.getInternal(Cursor.java:1284) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.Database.get(Database.java:1271) ~[je-7.3.7.jar:7.3.7]
        at com.sleepycat.je.Database.get(Database.java:1330) ~[je-7.3.7.jar:7.3.7]
        at com.starrocks.journal.bdbje.CloseSafeDatabase.get(CloseSafeDatabase.java:47) ~[starrocks-fe.jar:?]
        at com.starrocks.journal.bdbje.BDBJournalCursor.next(BDBJournalCursor.java:182) ~[starrocks-fe.jar:?]
        ... 3 more
2022-06-28 15:18:43,740 INFO (Thread-41|86) [StarRocksFE.lambda$addShutdownHook$1():364] start to execute shutdown hook
2022-06-28 15:18:45,346 INFO (nioEventLoopGroup-5-1|88) [RestBaseAction.handleRequest():55] receive http request. url=/api/bootstrap?cluster_id=756268253&token=c2535550-45d0-4def-98f2-ce181a9433fd
2022-06-28 15:18:46,704 INFO (Thread-41|86) [StarRocksFE.lambda$addShutdownHook$1():389] shutdown hook end
```
### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
 - MAIN-RELEASE-a4322aa

--- a/fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java
+++ b/fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java
@@ -1680,7 +1680,7 @@ public class GlobalStateMgr {
         listener.setMetaContext(metaContext);
     }
 
-    public synchronized boolean replayJournal(long toJournalId) {
+    public synchronized boolean replayJournal(long toJournalId) throws JournalException {
         long newToJournalId = toJournalId;
         if (newToJournalId == -1) {
             newToJournalId = getMaxJournalId();
@@ -1704,8 +1704,8 @@ public class GlobalStateMgr {
             JournalEntity entity = null;
             try {
                 entity = cursor.next();
-            } catch (InterruptedException | JournalException | JournalInconsistentException e) {
-                LOG.warn("got exception when get next, will exit, ", e);
+            } catch (InterruptedException | JournalInconsistentException e) {
+                LOG.warn("got interrupt exception or inconsistent exception when get next, will exit, ", e);
                 // TODO exit gracefully
                 Util.stdoutWithTime(e.getMessage());
                 System.exit(-1);

